src/lib/MailtrapClient.ts

```diff
@@ -58,15 +62,15 @@ export default class MailtrapClient {
     this.accountId = accountId;
 
     /**
-     * Initialize testing API.
+     * Initialize APIs.
      */
     this.testingAPI = new TestingAPI(
       this.axios,
       this.testInboxId,
       this.accountId
     );
-
     this.general = new GeneralAPI(this.axios, this.accountId);
+    this.bulkSending = new BulkSendingAPI(this.axios);
```

Comment:
Probably let's name it `bulk`? I.e. `general`, `testing`, and `bulk`.

Reply:
The name was taken from the documentation. However agree, looks much cleaner.